frontend-react/src/components/USLink.tsx

```diff
@@ -23,14 +23,14 @@ type USNavLinkProps = Pick<AnchorHTMLAttributes<{}>, "href"> & CustomLinkProps;
  * <USLink anchor href="#this-section-on-my-page">To Section</USLink> // uses <a>
  * */
 export const USLink = ({
-    anchor = false,
     children,
     className,
     href,
     state,
     ...anchorHTMLAttributes
 }: USLinkProps) => {
-    return !anchor ? (
+    const isAnchor = href?.startsWith("#");
+    return !isAnchor ? (
         <Link
             to={href || ""}
             className={classnames("usa-link", className)}
```

Comment:
Can the 'anchor' prop be removed on ln 75?

Reply:
Thanks @etanb and @penny-lischer for the call out. I thought I removed that prop from the interface and was expecting typescript errors as an indicator of its use to remove but I didn't. Will fix.